src/Identity/ApiAuthorization.IdentityServer/samples/ApiAuthSample/Startup.cs

```diff
@@ -50,6 +50,7 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             if (env.IsDevelopment())
             {
                 app.UseDeveloperExceptionPage();
+                app.UseMigrationsEndPoint();
```

Comment:
Alternatively, we could have the call to AddDatabaseDeveloperPageExceptionFilter add this middleware (conditioned on the development environment) in a startup filter. What do you think @Tratcher ?

Reply:
No, avoid adding middelware with a startup filter. Middleware can be very order sensitive.

Reply:
Yea that's what I thought as well, though I thought we might get away with migrations endpoint since it'll only be on in Development and might be benign if we add it towards the beginning of the middleware pipeline. It annoyed me that in order to achieve the function of UseDatabaseErrorPage, you'll now need three things: add the filter, UseDeveloperExceptionPage and UseMigrationsEndpoint hence why I was looking for more unorthodox ways to simplify this. I ultimately chose not to do it in this PR since I was concerned with what it can break.

Reply:
I agree with @Tratcher here